chesster.js

```diff
@@ -636,115 +547,39 @@ controller.hears([
     });
 });
 
-controller.hears([
+slack.hears(controller, [
```

Comment:
I definitely like the simplifications that slackjs provides on the `hear` functions. Alot of redundancy will be removed. Fantastic.